src/helpers/sqlalchemy_helpers.py

```diff
@@ -81,3 +89,31 @@ def get_query_handlers(self) -> List[QueryHandler]:
             QueryHandler("distinct", self.distinct_on_list, allow_empty_data=True),
         ]
         return query_handlers
+
+
+def many_to_many_table(first_table: str, second_table: str) -> Table:
+    def get_column(table_name: str) -> Column:
+        return Column(f"{table_name.lower()}_id", ForeignKey(f"{table_name.lower()}.id"), primary_key=True)
+
+    table = Table(
+        f"{first_table.lower()}_{second_table.lower()}",
+        BaseModel.metadata,
+        get_column(first_table),
+        get_column(second_table),
+    )
+    return table
+
+
+def snake_case_table_name(model_name: str) -> str:
+    table_name = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", model_name)
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", table_name).lower()
```

Comment:
Does this need to be done twice?

Reply:
yes this is necessary, but this current solution only supports table names of up to 3 words